### PR TITLE
Up-152-create notes endpoints

### PR DESCRIPTION
**Pull request by d-username**

### CODE REVIEW

**Comment by d-username** [comment]
#152
 In this PR I only added the CREATE NOTE, we are breaking it down to separate branches.

File: src/controllers/user.js [hunk 1]
```diff
@@ -136,3 +136,27 @@ export const updateProfile = async (req, res) => {
...
+
+export const createNote = async (req, res) => {
+  const teacherId = req.user.id
+  const studentId = Number(req.params.id)
+  const { content, isEditted } = req.body
+  const newNoteData = {
+    teacherId,
+    studentId,
+    content,
+    isEditted
```
Comment (dogezen) [inline src/controllers/user.js:148]: spelling? `isEdited`

File: src/controllers/user.js [hunk 2]
```diff
@@ -136,3 +136,27 @@ export const updateProfile = async (req, res) => {
...
+  }
+
+  try {
+    if (!content) {
```
Comment (dogezen) [inline src/controllers/user.js:152]: you're checking if content is provided

However, you should also check if req.user has permissions to create the note. How do you know the user is a TEACHER? you might be able to access this info via `req.user.type`

File: src/domain/note.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import dbClient from '../utils/dbClient.js'
+
+export default class Note {
+  static fromDb(note) {
+    return new Note(
+      note.content,
+      note.teacherId,
+      note.studentId,
+      note.isEditted,
```
Comment (dogezen) [inline src/domain/note.js:9]: spelling: `note.isEdited`

**Review by dogezen — CHANGES_REQUESTED** [review]

File: src/controllers/user.js (hunk 1 above)
Comment (d-username) [reply src/controllers/user.js:148]: its @Alex90Jennings :D italian spelling?
i am going to correct it now.

**Comment by d-username** [comment]
@dogezen ,  @Alex90Jennings  just corrected those concerns in his Branch., and he pushed his version.

**Review by dogezen — APPROVED** [review]

**Review by sigristarisa — APPROVED** [review]

**Pull request merged**